doctrine/custom_dql_functions.rst

```diff
@@ -69,4 +69,23 @@ In Symfony, you can register your custom DQL functions as follows:
             ),
         ));
 
+.. note::
+
+    In case the ``entity_managers`` were named explicitly, configuring the functions with the
+    orm directly will trigger the exception `Unrecognized option "dql" under "doctrine.orm"`.
+    The `dql` configuration block must be defined under the named entity manager.
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    doctrine:
+        orm:
+            # ...
+            entity_managers:
+                example_manager:
+                    # Place your functions here
+                    dql:
+                        datetime_functions:
+                            test_datetime: AppBundle\DQL\DatetimeFunction
```

Comment:
@rvanlaak Can you also add the config for the XML and PHP formats like we do it usually?